app/_components/TestComponent/BadComponent.tsx

```
@@ -0,0 +1,60 @@
+    const [inputValue, setInputValue] = useState(""); // Unused state variable
+
+    // Bad: This effect runs every render because dependencies are missing
+    useEffect(() => {
+        console.log("Component rendered");
+    });
+
+    function handleClick() {
+        setCounter(counter + 1); // Bad: Directly using counter may cause stale state issues
+    }
+
+    function handleClick() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'handleClick'. Consider to delete it or rename it.

'handleClick' is defined here:


(lint/suspicious/noRedeclare)